solana/programs/example-native-token-transfers/src/transfer.rs

```diff
@@ -0,0 +1,3 @@
+use ntt_messages::{ntt::EmptyPayload, ntt::NativeTokenTransfer};
+
+pub type NativeTokenTransferConcrete = NativeTokenTransfer<EmptyPayload>;
```

Comment:
naming nit: what's the rationale for calling this `Concrete`? Personally I think inlining the RHS everywhere would improve readability

Reply:
It’s a concrete type of the, now generic, NativeTokenTransfer. I was trying to avoid an integrator having to update a bunch of places in the code should they want to use an additional payload. I also figured this approach makes it safer, since it is unsafe (with the current implementation) to mix up the types used (where `EmptyPayload` currently is).

Reply:
I'm still not a fan of the name, because every instantiation is going to be concrete, so calling this one the concrete one is pretty arbitrary.
`NativeTokenTransferNoPayload` might be a better candidate.

This is cosmetic so let's do it in a follow up PR which should be easier to review than re-reviewing this one.

Reply:
https://github.com/wormhole-foundation/example-native-token-transfers/issues/526